index.md

```diff
@@ -1,21 +1,54 @@
 
 ## Links
-- Paper: [Arxiv](https://arxiv.org/abs/2305.15761){:target="_blank"}, [OpenReview](https://openreview.net/forum?id=cpT4zTZPAS){:target="_blank"}
-- Poster: [PDF](/resources/poster-rss-workshop-l4tamp-rsp-final.pdf){:target="_blank"}
-- Code (coming soon):
+- **Paper:** [Arxiv](https://arxiv.org/abs/2305.15761){:target="_blank"}, [OpenReview(2023 RSS workshop)](https://openreview.net/forum?id=cpT4zTZPAS){:target="_blank"}, [OpenReview(2024 ICRA workshop)](https://openreview.net/forum?id=DTu0hqvWaU){:target="_blank"}
+- **Poster:** [PDF](/resources/poster-rss-workshop-l4tamp-rsp-final.pdf){:target="_blank"}
+- **Code** (coming soon):
   - Source code: [MATLAB](https://www.github.com/ChirikjianLab/primp-matlab/){:target="_blank"}, [Python](https://www.github.com/ChirikjianLab/primp-python/){:target="_blank"}
   - Documentation
+- **Presentations:** Presented in RSS 2023 workshop on [Learning for Task and Motion Planning](https://zt-yang.github.io/rss23-l4tamp-workshop/){:target="_blank"}
+- **Supplementary Video:** [YouTube Chirikjian Lab](https://www.youtube.com/watch?v=yp1CCNiqw7w)
+ 
+
+## Features of PRIMP
+### Adaptation to new situations
+- Novel via-point poses with uncertainty
+<p align="center">
+  <img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/49382079-edd6-4c35-9731-4451c08c3799" alt="via point" width="400" />
+</p>
+
+- Change of viewing frame
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/dca9589d-1cbc-487c-9779-e7037fe1449d" alt="view frame" width="400" />
+</p>
+
+
+### Robot-agnostic
+- Skills can be easily transferred to another robot
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/49382079-edd6-4c35-9731-4451c08c3799" alt="robot agnostic" width="400" />
```

Comment:
The same figure with adaptation to via points?

Reply:
OK. I have changed the link, but I think I didn't successfully save them. I will update figure links.